apps/i18n/spritelab/en_us.json

```diff
@@ -318,6 +320,7 @@
   "thirdClickButNoPrint": "You clicked on more than two sprites but no text was printed. Add a `print` block under each `when clicked` event block.",
   "thirdClickButNoSpeech": "You clicked on more than two sprites but one of your clicks didn't cause a sprite to say anything. Add a `say` block under each `when clicked` event block. \n<xml><block type=\"gamelab_spriteSay\"><title name=\"SPEECH\">Hello, world!</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"default\"</title></block></value></block></xml>",
   "titleVariable": "Your title screen didn't include text from the prompt.",
+  "ToDo": "This feedback string needs to be added.",
```

Comment:
@onlinecsteacher Can we stick to the convention of having keys start with lower case letters? 

Separately, while I understand the purpose for this string, I'm hesitant to add more strings that our translators will have to translate if we know they're just temporary. Presumably, we'd never want a student to see this. This same file already contains some "generics" that I'd encourage you try to using instead:
https://github.com/code-dot-org/code-dot-org/blob/752a9ed715825719d9b64f1f078f8ca22361f701/apps/i18n/spritelab/en_us.json#L178-L184

Additionally, these strings, while not super helpful, are still at least a little student-friendly. If they happened to make it into a published level they'd maybe confusing but not look totally broken. Open to push-back here!

Reply:
I agree on this feedback. Try to avoid using generic keys if the string has a very specific use case, and use more contextual keys. I would use something like `addFeedbackStringMessage` if this string is use to show a message when a feedback string needs to be added.
If the message can be more generic feedback, stick to existing strings.